static/js/components/EducationForm.js

```diff
@@ -30,29 +32,29 @@ class EducationForm extends ProfileFormFields {
     setEducationDegreeInclusions:   React.PropTypes.func,
   };
 
-  openEditEducationForm = index => {
+  openEditEducationForm: Function = (index: number): void => {
     openEditEducationForm.call(this, index);
   };
 
-  openNewEducationForm = (level, index) => {
+  openNewEducationForm: Function = (level: string, index: number): void => {
     openNewEducationForm.call(this, level, index);
   };
 
-  deleteEducationEntry = () => {
+  deleteEducationEntry: Function = (): void => {
     deleteEducationEntry.call(this);
   };
 
-  renderEducationLevel(level){
+  renderEducationLevel(level: {value: string; label: string}) {
```

Comment:
They have some examples in the object type section of the docs: https://flowtype.org/docs/objects.html#_

mainly trial and error though :P